ezcrypto.gemspec

```diff
@@ -1,12 +1,19 @@
+# frozen_string_literal: true
+
+lib = File.expand_path('lib', __dir__)
+$LOAD_PATH.unshift(lib) unless $LOAD_PATH.include?(lib)
+require 'ezcrypto/version'
+
 Gem::Specification.new do |s|
   s.name        = 'ezcrypto'
-  s.version     = '0.7.2'
-  s.date        = '2009-03-10'
+  s.version     = EzCrypto::VERSION
+  s.date        = '2019-05-31'
```

Comment:
is this usually hardcoded in our other gems?

Reply:
I could not get gem bump to work without a version.rb file.
af_crypto, af_encrypted_attr, af_email all have version.rb.

Reply:
yep. That is a requirement for AfGem pretty much. I was talking about the date though :-)

Reply:
Oh. I removed the date, it's pointless